### ISSUE TITLE
Should this get refactored with React?

### ISSUE THREAD

**Issue by ghostlandr** [holtscomm/miniblog#48]
There isn't a great need for two-way data-binding here. Plus it'd be a good opportunity to use React.

**Comment by ghostlandr** [comment]
I've started this work. It'll solve a problem I had with the Knockout version right off the hop, which is that it's able to internalize the post template. In theory it'll still be relatively simple to customize it from the user's end, but I guess that remains to be seen.

**Comment by ghostlandr** [comment]
Well, and that is to say that it will be incredibly simple to customize if they're comfortable in code.

**Comment by gholtslander** [comment]
Which will mean almost zero % of users will be comfortable customizing. 

—
Gord

**Comment by ghostlandr** [comment]
At least at this point I think of this as a tool some dev would use to drop a blog in somewhere. The install process isn't very user-friendly at this point either.

**Comment by gholtslander** [comment]
Yes, from a dev view, I'd agree the adjustments are pretty straightforward,
in that even I can often see what needs doing. However, I'm thinking that
if it's common folk 'running' the blog, then this will be a very high bar.

On Thu, Jul 2, 2015 at 8:22 AM, Graham Holtslander <notifications@github.com

> wrote:
> 
> At least at this point I think of this as a tool some dev would use to
> drop a blog in somewhere. The install process isn't very user-friendly at
> this point either.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/holtscomm/miniblog/issues/48#issuecomment-118048837.

## 

## 

Gord
Gordon Holtslander, BEd
Partner, Holtslander Communications
holtslandercommunications.ca
http://t.sidekickopen07.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XX45vg7PzW63Bmnv63K11RW5wvxsz56dVcJf1LcZyz02?t=http%3A%2F%2Fholtslandercommunications.ca%2F&si=4947150685863936&pi=cf3367ac-c05b-4cdc-c0f5-d9ad1a53a12c
gordon@holtslander.com

**Comment by ghostlandr** [comment]
Agreed. Part of the effort in all of this has to be user-friendliness; right now it would be difficult for said common folk to run the blog.

**Comment by ghostlandr** [comment]
I think now that I've been doing the zu thing for a while I'm a lot better equipped to understand and carry out what's involved in making this a simpler install. Looking forward to it!
